src/Components/Dictionary/WordTerm.js

```diff
@@ -1,37 +1,75 @@
+/* eslint-disable react/button-has-type */
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useState } from 'react';
 import {
-  Card, CardActions, CardContent, Button, Typography,
+  Card, CardActions, CardContent, Button, Grid, Avatar,
 } from '@mui/material';
+import './WordTerm.css';
 
-export default function WordTerm({ word, def, links }) {
+export default function WordTerm({
+  word, def, links, wordBreakdown,
+}) {
+  const [showMore, setShowMore] = useState(false);
   const linksArray = links.split('\n');
   if (linksArray.length !== 0) { linksArray.pop(); }
   return (
-    <Card sx={{ minWidth: 275 }}>
+    <Card sx={{
+      borderRadius: 8, margin: 4, boxShadow: 5, marginRight: 20, marginLeft: 20,
```

Comment:
I think the padding around/within the cards could use some adjustment! Rather than using constant numbers, we can use percentages to make the card size adjust with the screen!